tests/cds-rdm/test_full_migration.py

```diff
@@ -133,10 +135,22 @@ def orcid_id(record):
                 "name": "Mendoza, Diego",
                 "given_name": "Diego",
                 "family_name": "Mendoza",
-                "identifiers": [{"identifier": "2773374", "scheme": "lcds"}],
+                "identifiers": [
+                    {"identifier": "2773374", "scheme": "lcds"},
+                    {"identifier": "0009-0007-7638-4652", "scheme": "orcid"},
+                ],
             }
         }
     ]
+    lcds_identifier = dict_rec["metadata"]["creators"][0]["person_or_org"][
+        "identifiers"
+    ][0]
+    orcid_identifier = orcid_name_data["identifiers"][0]
+
+    name_from_db = NamesMetadata.query.filter_by(pid=orcid_name_data["id"]).first()
+    assert "identifiers" in name_from_db.json
+    assert orcid_identifier in name_from_db.json["identifiers"]
+    assert lcds_identifier in name_from_db.json["identifiers"]
```

Comment:
👌 thanks for fixing it! 
could you add a check to make sure person id will not end up inside the record?
it would be even better for the future if you check if `UserIdentity` with id = person id was created, and RemoteAccount object with migration info stored in extra_data was also done (check the transform.py code for the owner of the record field - you will find more details there)